tests/foreman/cli/test_errata.py

```diff
@@ -1390,9 +1448,116 @@ def test_positive_user_permission(self):
         @Assert: Check that the new user is able to see errata for one
         product only.
 
-        @caseautomation: notautomated
-
         """
+        user_password = gen_alphanumeric()
+        user_name = gen_alphanumeric()
+        org = self.org3
```

Comment:
@svtkachenko it leeds to the same when using orgs[0], orgs[1], orgs[3]
orgs[3].products[0], orgs[3].products[1]